browser/src/components/LogView/BranchGraph/index.tsx

```diff
@@ -274,11 +317,17 @@ function drawGitGraph(svg: SVGSVGElement, content: HTMLElement, startAt: number,
         svgPaths.set(branch.path, svgPath);
     });
 
-    debugger;
     const lines: Point[][] = [];
     svgPaths.forEach((pathOrSvg, svg) => {
         try {
-            lines.push(getPointsFromPath(pathOrSvg));
+            const points = getPointsFromPath(pathOrSvg);
+            lines.push(points);
+
+            // Re-generate the paths with smooth curvy edges.
+            const pathGenerator = new PathGenerator();
+            points.forEach(point => pathGenerator.addPoint(point));
+            svg.setAttribute('d', pathGenerator.path);
```

Comment:
@ole1986 
Again this is a hacky but safe approach.
I'm parsing the current path and re-building it.
The proper way would be to build the path once, but that requires a cleanup and I'd rather do that separately.